google_scraper.py

```diff
@@ -155,38 +161,41 @@ def download_image(link, image_data):
     links = [json.loads(i.text)["ou"]
              for i in soup.find_all("div", class_="rg_meta")]
     print("[%] Indexed {} Images.".format(len(links)))
+
     print("\n===============================================\n")
-    print("[%] Getting Image Information.")
+    print("[%] Getting image metadata\n")
     images = {}
     linkcounter = 0
+    errorcounter = 0
     for a in soup.find_all("div", class_="rg_meta"):
-        print("\n------------------------------------------")
         rg_meta = json.loads(a.text)
         if 'st' in rg_meta:
             title = rg_meta['st']
         else:
             title = ""
         link = rg_meta["ou"]
-        print("\n[%] Getting info on: {}".format(link))
+        print("==> Getting info on: {}".format(link))
         try:
             image_data = "google", query, rg_meta["pt"], rg_meta["s"], title, link, rg_meta["ru"]
             images[link] = image_data
         except Exception as e:
             images[link] = image_data
-            print("[!] Issue getting data: {}\n[!] Error: {}".format(rg_meta, e))
+            print("==> Issue getting data: {}\n[!] Error: {}".format(rg_meta, e))
+            errorcounter += 1
 
+        print("==> Information received for: {}".format(link))
         linkcounter += 1
 
     # Open i processes to download
-    print("\n------------------------------------------\n")
+    print ("\n Done fetching metadata.  {} succeeded | {} failed".format(linkcounter,errorcounter))
     print("\n===============================================\n")
     download_image.delta = 0
+    download_image.errors = 0
     for i, (link) in enumerate(links):
-        print("\n------------------------------------------\n")
         try:
             download_image(link, images[link])
         except Exception as e:
             error(link)
 
-    print("\n\n[%] Done. Downloaded {} images.".format(download_image.delta))
+    print("\n\n[%] Done. Downloaded images.   {} succeeded | {} failed".format(download_image.delta,download_image.errors))
```

Comment:
`print("\n\n[%] Done. Downloaded {} images. Failed to download {} images.".format(download_image.delta, download_image.errors))`